src/app/redux/reducers/audiochallengeReducer.ts

```diff
@@ -0,0 +1,137 @@
+import { createReducer, on, Action } from '@ngrx/store';
+import { IAudiochallengeWord } from 'src/app/redux/models/IAudiochallengeWord';
+import { IWord } from 'src/app/core/models/IWord';
+import {
+  wordsLoadedSuccess,
+  audiochallengeStarted,
+  audiochallengeEnded,
+  translationChoosed,
+  showNextWord,
+  translationsLoadedSuccess,
+  rightAnswer,
+  wrongAnswer,
+  gameOver,
+  soundOf,
+  soundOn,
+  translationsShuffled,
+  closeGame,
+} from '../actions/audiochallenge.actions';
+import { AudiochallengeState } from '../models/audiochallenge.state.model';
+import { ASSETS_API_URL } from 'src/app/core/constants/mini-games';
+import { FAIL_AUDIO_URL, SUCCESS_AUDIO_URL } from 'src/app/core/constants/audiochallenge-game';
+
+const initialWord: IAudiochallengeWord = {
+  id: '',
+  word: '',
+  image: '',
+  audio: '',
+  audioMeaning: '',
+  audioExample: '',
+  textMeaning: '',
+  textExample: '',
+  transcription: '',
+  wordTranslate: '',
+  textMeaningTranslate: '',
+  textExampleTranslate: '',
+  translationsArray: [],
+};
+
+export const initialState: AudiochallengeState = {
+  list: [],
+  statsList: [],
+  isGameStarted: false,
+  isGameEnded: false,
+  currentWord: initialWord,
+  audio: new Audio(),
```

Comment:
It's not a good idea to keep this (audio) in reducers

Reply:
Done